src/components/controls/TextArea/__tests__/TextArea.test.tsx

```diff
@@ -23,6 +23,16 @@ describe('TextArea', () => {
             expect(screen.getByText('Some Error')).toBeVisible();
         });
 
+        test('render note container with note prop', () => {
+            const {container} = render(
+                <TextArea error="Some Error" note={<div>Additional</div>} />,
+            );
+
+            // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
+            expect(container.querySelector('.g-text-area__note')).toBeInTheDocument();
```

Comment:
Why check for element existence? Isn't it the same element that you check next?

Reply:
Did the same test with the error container

<img width="796" alt="image" src="https://github.com/gravity-ui/uikit/assets/27211707/eac289e3-232c-4b4f-a321-a2f1a0e279ee">

Reply:
I think getByText is quite enough